### PR TITLE
Minubo integration initial release

### PR DESCRIPTION
**Pull request by valerio8787**
- Developer(s): @valerio8787

- Peer Reviewer:

- Ticket: https://spryker.atlassian.net/projects/ECO/issues/ECO-1259

- Release: https://release.spryker.com/release-groups/view/60

#### Please confirm

- [ ] No new OS components - or they have been approved by legal department
- [ ] All new or heavily changed classes get an "A" rating (Scrutinizer), except Facades, Factories, QueryContainers and Tests

#### Documentation

- [ ] Functional documentation provided or in progress?
- [ ] Integration guide for projects provided or not needed?
- [ ] Migration guides for all contained majors provided or not needed?

#### Release Table

   Module                | Release Type         | Constraint Updates         |
   :--------------------- | :------------------------ | :--------------------- |
   Minubo                | major (new)           | Oms 8.1.0                      |

#### Release Notes

Initial Release of Minubo Module. Module provides functionality to export order and customer data that was updated since last run of export. Data exported to Amazon S3 bucket as file with list of JSON-objects.

-----------------------------------------

#### Module Minubo

_**Maxi-Major:** New or fully refactored module._

Def of done (by responsible developer):
- [ ] All methods in facade, client and plugin interfaces provide an [API-doc](https://academy.spryker.com/understanding_spryker/definition_api.html).
- [ ] All [dependencies](https://spryker.atlassian.net/wiki/spaces/CORE/pages/4980746/Module+Dependency+Guidelines) are defined, checked and corrected.
- [ ] There are no violations of the architecture rules. All [cleanups](https://release.spryker.com/module-cleanups) checked.
- [ ] All new or changed business logic is covered by functional tests (in Zed business layer).
- [ ] Migration guide provided (for existing modules).
- [ ] New modules have module description in README, for existing ones they need to be up to date.

##### Change log

Initial Release

Module provides functionality to export order and customer data that was updated since last run of export. Data exported to Amazon S3 bucket as file with list of JSON-objects.

### CODE REVIEW

**Review by FabianWesner — APPROVED** [review]
Well done!

**Comment by dereuromark** [comment]
"Initial Release of Minubo Module"
should actually read what this is about IMO. A bit more high level details.

I fixed up the template regarding the module.

File: src/SprykerEco/Shared/Minubo/MinuboConstants.php [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Shared\Minubo;
+
+interface MinuboConstants
+{
+    const MINUBO_FILE_SYSTEM_NAME = 'MINUBO:MINUBO_FILE_SYSTEM_NAME';
```
Comment (dereuromark) [inline src/SprykerEco/Shared/Minubo/MinuboConstants.php]: New PHP7 demands a visibility.
Is this protected or public? Looks like the latter to me.

File: src/SprykerEco/Zed/Minubo/Business/Exporter/CustomerDataExporter.php [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+use SprykerEco\Zed\Minubo\Persistence\MinuboRepositoryInterface;
+
+class CustomerDataExporter implements DataExporterInterface
+{
+    const FILE_PREFIX = 'Customers';
+
+    /**
+     * @var \SprykerEco\Zed\Minubo\Persistence\MinuboRepositoryInterface
+     */
+    private $repository;
```
Comment (dereuromark) [inline src/SprykerEco/Zed/Minubo/Business/Exporter/CustomerDataExporter.php]: protected would suffice IMO.

File: src/SprykerEco/Zed/Minubo/Business/Exporter/OrderDataExporter.php [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+use SprykerEco\Zed\Minubo\Persistence\MinuboRepositoryInterface;
+
+class OrderDataExporter implements DataExporterInterface
+{
+    const FILE_PREFIX = 'Orders';
+
+    /**
+     * @var \SprykerEco\Zed\Minubo\Persistence\MinuboRepositoryInterface
+     */
+    private $repository;
```
Comment (dereuromark) [inline src/SprykerEco/Zed/Minubo/Business/Exporter/OrderDataExporter.php]: Same here

File: tests/SprykerEcoTest/Zed/Minubo/Business/MinuboFacadeTest.php [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+use Codeception\TestCase\Test;
+
+class MinuboFacadeTest extends Test
+{
+    /**
+     * @return void
+     */
+    public function testMinuboFacade(): void
+    {
+
```
Comment (dereuromark) [inline tests/SprykerEcoTest/Zed/Minubo/Business/MinuboFacadeTest.php]: Looks like the test was never added?

File: src/SprykerEco/Shared/Minubo/MinuboConstants.php (hunk 1 above)
Comment (valerio8787) [reply src/SprykerEco/Shared/Minubo/MinuboConstants.php]: 👍

**Comment by dereuromark** [comment]
We cannot release on a non-master branch though.
Why doesnt this target "master"?

**Pull request merged**